lib/stacks/dynamodb-stack.ts

```diff
@@ -0,0 +1,36 @@
+import { GlobalSecondaryIndexProps, AttributeType } from "@aws-cdk/aws-dynamodb";
+import { App, Stack, StackProps } from "@serverless-stack/resources";
+import { DynamoDBTable, DynamoDbTableProps } from "../constructs/dynamodb/dynamodb-table";
+
+const pushTableGSIs: Array<GlobalSecondaryIndexProps> = [
```

Comment:
My DynamoDBTable class  works differently than how you have it here. On main the DynamoDBTable class is a stack that creates each ofthe dynamo db tables...  I think your approach makes more sense so we can merge in what you have

Reply:
I saw yours and this was a modification to extract the base dynamo table into its own class and use the stack to create instances of it. I was thinking this way it would be easier to make general DynamoDB changes that get applied to every table and we don't end up with a giant DynamoDB stack file.